### PR TITLE
content changes

### PR DESCRIPTION
**Pull request by abdolg**

### CODE REVIEW

File: data/team.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   },
 
   {
-    "name": "Felix Hofmann",
```
Comment (georgiee) [inline data/team.json:23]: 👍

File: src/assets/locales/de/translations.json [hunk 2]
```diff
@@ -2,17 +2,17 @@
   "main.description": "Wir machen die digitale Welt einfacher. Mit maßgeschneiderter Web-Software für Unternehmen. Von Frontend bis Backend. Vom Webauftritt bis zur digitalen Sicherheitsinfrastruktur.",
   "main.services.title": "Wir verbinden. Mit Technologie.",
   "main.services.kicker": "Unsere Leistungen",
-  "main.services.text": "Unsere Software Ingenieure und Designer vereint eines: die Liebe zum Detail und zur Technologie. Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität. Und jeder Menge Herzblut.",
+  "main.services.text": "Unsere Softwareingenieur:innen und Designer:innen vereint eines: die Liebe zum Detail und zur Technologie. Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität. Und jeder Menge Herzblut.",
```
Comment (georgiee) [inline src/assets/locales/de/translations.json:5]: 👍

File: src/assets/locales/de/translations.json [hunk 3]
```diff
@@ -39,46 +39,46 @@
...
-  "career.application-process.accordion.0.title": "1. Bewerb dich",
-  "career.application-process.accordion.0.paragraph": "Alles beginnt mit deiner Bewerbung. Schicke uns deinen Lebenslauf und ein kleines Anschreiben, wieso du dich gerade bei uns bewirbst. In deiner Bewerbung achten wir nicht auf Schulnoten, sondern suchen nach deiner Leidenschaft fürs Coden. Zeig uns deinen GitHub oder GitLab Account. Hast du einen Blog? Dann verweise gerne darauf.",
+  "career.application-process.paragraph": "Sich auf einen Job zu bewerben, ist oft anstrengend. Wir kennen das. Deshalb hat sich unser Team hat sich vier Schritte überlegt, die alles einfacher machen.",
+  "career.application-process.accordion.0.title": "1. Bewirb dich",
+  "career.application-process.accordion.0.paragraph": "Schicke uns deinen Lebenslauf und ein kurzes Anschreiben, in dem Du uns erzählst, wieso Du dich gerade bei uns bewirbst. Dabei ist uns nicht dein Abschluss, sondern  deine Leidenschaft fürs Coden wichtig. Zeig uns deinen GitHub- oder GitLab-Account, deinen Blog oder YouTube-Kanal. Ganz egal. Hauptsache, wir spüren, wofür Du brennst.",
```
Comment (georgiee) [inline src/assets/locales/de/translations.json]: so much better 💪

File: src/assets/locales/de/translations.json [hunk 4]
```diff
@@ -39,46 +39,46 @@
...
-  "career.perk.teaser.3.title": "Arbeiten wo du willst",
-  "career.perk.teaser.3.description": "Arbeitszeit und Arbeitsort kannst du nach Absprache mit dem Team & Kunden flexibel wählen.",
+  "career.perk.teaser.2.description": "Ob MacBook oder Linux Laptop. Ob Maus, Trackpad, VSCode oder IntelliJ. Du hast bei uns die Wahl.",
+  "career.perk.teaser.3.title": "Arbeiten wo Du willst",
+  "career.perk.teaser.3.description": "Arbeitszeit und Arbeitsort kannst Du in Absprache mit deinem Team und Kunden flexibel wählen.",
   "career.perk.teaser.4.title": "Konferenzen",
-  "career.perk.teaser.4.description": "Wir lieben Tech-Konferenzen und sind schon oft Besucher und manchmal sogar Speaker gewesen. Wir zahlen dir Ticket, Kost und Logis. Willst du Speaker werden? Wir unterstützen dich dabei.",
+  "career.perk.teaser.4.description": "Wir lieben Tech-Konferenzen und sind schon bei vielen gewesen – manchmal sogar als Speaker. Das wollen wir auch für dich: Deshalb zahlen wir zahlen dir Ticket, Kost und Logis – und unterstützen dich, wenn Du Ambitionen hast, Speaker:in zu werden.",
```
Comment (georgiee) [inline src/assets/locales/de/translations.json]: > Deshalb zahlen wir zahlen dir Ticket
something is wrong

File: src/assets/locales/de/translations.json [hunk 5]
```diff
@@ -111,47 +111,48 @@
   "about-us.office.title": "Office",
   "about-us.office.kicker": "Satellytes",
   "about-us.office.heading": "Im Herzen von München - direkt am Sendlinger Tor",
-  "about-us.office.text": "Fühlen Sie sich bei uns wie zu Hause. Ein paar kleine Eindrücke unseres neuen Büros in der Sendlinger Straße im Herzen Münchens.",
+  "about-us.office.text": "Unser Büro ist ein Ort zum Arbeiten, Wohlfühlen, Kennenlernen und Zusammenkommen. Für uns und unsere Kund:innen.",
```
Comment (georgiee) [inline src/assets/locales/de/translations.json:114]: 👍

File: src/assets/locales/de/translations.json [hunk 6]
```diff
@@ -111,47 +111,48 @@
...
   "about-us.team.title": "Team",
   "about-us.team.heading": "Das sind wir",
-  "about-us.team.text": "Unser erfahrenes Team aus Software Ingenieuren und Designern",
+  "about-us.team.text": "Unser erfahrenes Team aus Softwareingenieur:innen und Designer:innen.",
```
Comment (georgiee) [inline src/assets/locales/de/translations.json:117]: great that we handle the gender topic properly now.

File: src/assets/locales/de/translations.json [hunk 7]
```diff
@@ -111,47 +111,48 @@
...
+  "services.kicker": "Von der Plattform bis zum Produkt",
+  "services.text": "Unsere Liebe für exzellente Technologie ist genau so groß wie unsere Liebe für gutes Design. Deshalb vereinen wir beide Welten. Damit Sie sich nicht nur auf Funktionalität und Sicherheit verlassen können. Sondern auch darauf, dass Ihre Kunden ein digitales Erlebnis geboten bekommen.",
   "services.platform.title": "Digitale Plattformen",
-  "services.platform.teaser": "Von low-code bis zu Designsystemen.",
-  "services.platform.text": "Wir sind Spezialisten für digitale Plattformen von der Idee bis zur\n          Umsetzung. Unsere langjährige Erfahrung bei der Gestaltung von B2B-\n          und B2C-Plattformen aus den unterschiedlichsten Branchen macht uns zum\n          idealen Partner für Ihr Vorhaben.",
+  "services.platform.teaser": "Ob B2B- und B2C-Plattform – wir sind der ideale Partner für Sie.",
+  "services.platform.text": "Wir sind Spezialist:innen für digitale Plattformen. Von der Idee bis zur Umsetzung. Unsere langjährige Erfahrung bei der Gestaltung von B2B- und B2C-Plattformen aus den unterschiedlichsten Branchen macht uns zum idealen Partner für Ihr Projekt.",
```
Comment (georgiee) [inline src/assets/locales/de/translations.json:127]: shorter sentences. I like that.

File: src/assets/locales/de/translations.json [hunk 8]
```diff
@@ -111,47 +111,48 @@
...
   "services.platform.list.platforms": "Digital platform design & development",
   "services.platform.list.lowcode": "Low-Code Systems",
   "services.platform.list.designsystems": "Design Systeme",
   "services.platform.list.infrastructure": "Infrastruktur & Automation",
   "services.consulting.title": "Beratung",
-  "services.consulting.teaser": "Die langjährige Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz.",
-  "services.consulting.text": "Die langjährige Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz. Wir\n          teilen unser Wissen sehr gerne mit Ihnen und beraten Sie in technischen und\n          konzeptionellen Angelegenheiten.",
+  "services.consulting.teaser": "Sie können sich auf unsere langjährige Erfahrung verlassen.",
```
Comment (georgiee) [inline src/assets/locales/de/translations.json:133]: that's less drama in the sentence 😄

File: src/components/pages/landingpage/service.tsx [hunk 9]
```diff
@@ -32,18 +32,18 @@ export const Service = () => {
...
         >
-          {t('services.consulting.teaser')}
+          {t('services.products_services.teaser')}
         </Teaser>
         <Teaser
-          title={t('services.products_services.title')}
+          title={t('services.consulting.title')}
           linkTo="/services"
-          illustration="book_038"
+          illustration="scientist_042"
```
Comment (georgiee) [inline src/components/pages/landingpage/service.tsx:44]: oh some wrong keys and icons?

File: src/components/pages/service/service.tsx [hunk 10]
```diff
@@ -28,7 +28,11 @@ export const Service = () => {
       transparentHeader={true}
       light={true}
       leadbox={leadbox}
-      hero={<AuroraHero title={t('services.hero')} />}
+      hero={
+        <AuroraHero title={t('services.hero')}>
+          {t('services.hero.description')}{' '}
```
Comment (georgiee) [inline src/components/pages/service/service.tsx:33]: we never had a text here and reused the title. You added a text, is that correct?

File: src/assets/locales/de/translations.json [hunk 11]
```diff
@@ -39,46 +39,46 @@
...
   "career.application-process.accordion.2.title": "3. Technisches Interview",
-  "career.application-process.accordion.2.paragraph": "Wir machen weder Whiteboard Interviews noch schicken wir die irgendwelche Programmieraufgaben. Stattdessen wirst du vorab ein Code Review durchführen, das wir dann zusammen im Interview durchsprechen. Hier werden sich ganz automatisch spannende Gespräche ergeben. Wir werden Probleme im Code besprechen und das ein oder andere Fachwissen über relevante Technologien ansprechen. Neben der ganzen Technik haben wir auch Zeit etwas über unsere Kultur bei Satellytes zu sprechen, um sicherzustellen, dass du dich bei uns wohl fühlst.",
+  "career.application-process.accordion.2.paragraph": "Wir machen weder Whiteboard Interviews noch schicken wir dir irgendwelche Programmieraufgaben. Stattdessen möchten wir, dass du ein Code Review machst, das wir im Interview durchsprechen. Dabei ergeben sich ganz automatisch spannende Gespräche – über Probleme im Code, Fachwissen und relevante Technologien. Und natürlich haben wir neben der ganzen Technik auch Zeit, über unsere Kultur zu sprechen. Um sicherzustellen, dass Du wir zu dir passen. Und du zu uns.",
```
Comment (georgiee) [inline src/assets/locales/de/translations.json]: something is off with the sentence

>  dass Du wir zu dir passe

**Review by georgiee — COMMENTED** [review]
Thanks for the great update! See my comments. once fixed we can approve and merge.
Regarding your PR: Next time please provide a nice description and guide the reviewer through some pages with links and notes. This makes the review much more focused. Here I just went through the PR sentences without a lot of emotion. In addition I identified the page parts to see it in action. Help me the next time. Thanks!

File: src/assets/locales/de/translations.json (hunk 11 above)
Comment (abdolg) [reply src/assets/locales/de/translations.json]: fixed

File: src/components/pages/service/service.tsx (hunk 10 above)
Comment (abdolg) [reply src/components/pages/service/service.tsx:33]: No we didn't but now we have one. I wanted some content on the service page as well. And yes it works fine -> https://satellytescommain21751-updatetext.gtsb.io/de/services/

Comment (georgiee) [reply src/components/pages/service/service.tsx:33]: yeah that's good, thanks!

**Review by georgiee — APPROVED** [review]
Let's go with those changes 🙌

**Pull request merged**